Resources/string.csv

```diff
@@ -11,735 +20,734 @@
+KillButtonText,击杀,Kill,Убить
+SniperSnipeButtonText,狙击,Snipe,Застрелить
+FireWorksExplosionButtonText,引爆,Fire,Зажечь
+FireWorksInstallAtionButtonText,设置,Set,Установить
+SerialKillerSuicideButtonText,自杀倒计时,Suicide Timer,До Суицида
+WarlockCurseButtonText,下咒,Curse,Проклинать
+AssassinKillButtonText,刺杀,Kill,Убить
+AssassinMarkButtonText,标记,Mark,Пометить
+WitchSpellButtonText,诅咒,Spell,Заклинать
+VampireBiteButtonText,吸血,Bite,Укусить
+MinerTeleButtonText,传送,Deliver,Отправиться
+ArsonistDouseButtonText,涂油,Douse,Облить
+PuppeteerOperateButtonText,操控,Manipulate,Управлять
+BountyHunterChangeButtonText,变更,Swap,До Смены
+EvilTrackerChangeButtonText,追踪,Track,Отслеживать
+DefaultShapeshiftText,变形,Shift,Превращение
+DisabledBySettings,已在设置被中禁用,Disabled by Settings,Отключено настройкой
+Disabled,禁用,Disabled,Отключено
+FailToTrack,无法追踪,Failed To Track,Не Удалось Отследить
 
 ## 作弊
-Cheat.AUM,开挂（使用AUM）
+Cheat.AUM,开挂（使用AUM）,Hang up (using AUM),Повесить трубку (используя AUM)
 
 ## 驱逐确认
-PlayerIsRole,{0} 是 {1}
-PlayerExiled,{0} 被驱逐了...
-ExiledJester,票出 {0} 让你有种不详的预感\n{1} 在狂笑着...
-ExiledExeTarget,票出 {0} 让你有种不详的预感\n他是 {1}，而他的背后传来处刑者阴沉的笑声...
-NoImpRemain,已经没有内鬼啦
-ImpRemain,剩余 {0} 个内鬼
-NeutralRemain,剩余 {0} 个邪恶中立
-IsGood,{0} 是个好人
+PlayerIsRole,{0} 是 {1},{0} is {1},{0} имеет роль {1}
+PlayerExiled,{0} 被驱逐了...,{0} was ejected...,{0} был изгнан...
+ExiledJester,票出 {0} 让你有种不详的预感\n{1} 在狂笑着...,Ticket {0} gives you a bad feeling \nbut {1} laughs wildly...,Билет {0} вызывает у вас дурное предчувствие\nно {1} дико смеется...
+ExiledExeTarget,票出 {0} 让你有种不详的预感\n他是 {1}，而他的背后传来处刑者阴沉的笑声...,"The ticket {0} gives you a bad feeling \nit is {1}, and behind it the Executioner's gloomy laughter sounds...","Билет {0} вызывает у вас дурное предчувствие \nЭто {1}, а позади него звучит мрачный смех Палача..."
+NoImpRemain,已经没有内鬼啦,There are no more impostors left,Предателей больше не осталось
+ImpRemain,剩余 {0} 个内鬼,There are {0} Impostors left,Осталось {0} Предателя
+NeutralRemain,剩余 {0} 个邪恶中立,There Are {0} Evil Neutrals Left,Осталось {0} Злых Нейтралов
+IsGood,{0} 是个好人,{0} nice guy,{0} хороший парень
 
 ## 游戏结束原因
-GameOverReason.HumansByTask,船员完成了任务
-GameOverReason.HumansByVote,内鬼被驱逐
-GameOverReason.HumansDisconnect,船员断线
-GameOverReason.ImpostorByKill,内鬼人数占优
-GameOverReason.ImpostorBySabotage,内鬼破坏了设施
-GameOverReason.ImpostorByVote,船员被驱逐
-GameOverReason.ImpostorDisconnect,内鬼断线
\ No newline at end of file
+GameOverReason.HumansByTask,船员完成了任务,The Crewmates completed the tasks,Члены Экипажа выполнили задания
+GameOverReason.HumansByVote,内鬼被驱逐,The Crewmates were expelled,Члены Экипажа были изгнаны
+GameOverReason.HumansDisconnect,船员断线,Crewmates disconnected,Члены Экипажа вышли из игры
+GameOverReason.ImpostorByKill,内鬼人数占优,The Impostor killed everyone,Предатели убили всех
+GameOverReason.ImpostorBySabotage,内鬼破坏了设施,The Impostors sabotaged the Crewmates,Предатели саботировали Членов Экипажей
+GameOverReason.ImpostorByVote,船员被驱逐,The Impostors was ejected by vote,Предатели были изгнаны

```

Comment:
Here is The Crewmates were expelled

Reply:
yeah that's right
and `GameOverReason.ImpostorByVote` means Imp win the game by vote
so it shoult translate in `The Crewmates were expelled`